frontend/src/components/widgets/NumberInput/NumberInput.tsx

```diff
@@ -228,18 +228,18 @@ class NumberInput extends React.PureComponent<Props, State> {
             }}
           />
           <div className="controls">
-            <span
+            <button
               className="control step-down"
               onClick={this.modifyValueUsingStep("decrement")}
             >
               <Icon type="minus" />
-            </span>
-            <span
+            </button>
+            <button
```

Comment:
This is not a requirement for this PR, just an idea: why not use a BaseWeb Button here?

This way the button's "focus" and "hover" UX state would better match the UX we already have for other BaseWeb Buttons. 

Although we'd have to update this button's theme, since our BaseWeb buttons look different from these. (White instead of gray, etc)

If this is too much work, no need to do it. But please at least copy the focus state from our BaseWeb widgets.

Reply:
I've changed the focus to get the same style as the hover, i changed the span to button due to the doble click select problem, the problem is if we use the BaseWeb Buttons, we are gonna need to overried the styles to match the feeling of the input as we are doing with the borders and background color, that's why i did not use the baseweb component